src/RESTController.js

```diff
@@ -38,6 +38,58 @@ if (typeof XMLHttpRequest !== 'undefined') {
 if (process.env.PARSE_BUILD === 'node') {
   XHR = require('xmlhttprequest').XMLHttpRequest;
 }
+if (process.env.PARSE_BUILD === 'weapp') {
+  class WeappXhr {
```

Comment:
Move this to a new file `WeAppXHR.js`? 
Why can't you use XHR? Are there some restrictions?

Reply:
The Wechat miniprogram does not support XHR

Reply:
Do queries etc work with this wx.request? If so you can ignore my comment.

Reply:
I see they have a web socket api, you could implement that as well. Do files upload correctly?

Reply:
Supporting queries, socket with my support when I have time

Reply:
Livequery has been supported